translator/translate/otel/pipeline/host/translators.go

```diff
@@ -36,11 +42,33 @@ func NewTranslators(conf *confmap.Conf, os string) (pipeline.TranslatorMap, erro
 	hasHostPipeline := hostReceivers.Len() != 0
 	hasDeltaPipeline := deltaReceivers.Len() != 0
 
-	if hasHostPipeline {
-		translators.Set(NewTranslator(common.PipelineNameHost, hostReceivers))
+	otherExporters := map[string]common.Translator[component.Config]{
+		common.AMPKey: prometheusremotewrite.NewTranslatorWithName("amp"),
+	}
+
+	if !conf.IsSet(metricsDestinationsKey) || conf.IsSet(common.ConfigKey(metricsDestinationsKey, "cloudwatch")) {
+		exporters := common.NewTranslatorMap(awscloudwatch.NewTranslator())
+		if hasHostPipeline {
+			translators.Set(NewTranslator(common.PipelineNameHost, hostReceivers, exporters))
+		}
+		if hasDeltaPipeline {
+			translators.Set(NewTranslator(common.PipelineNameHostDeltaMetrics, deltaReceivers, exporters))
+		}
 	}
-	if hasDeltaPipeline {
-		translators.Set(NewTranslator(common.PipelineNameHostDeltaMetrics, deltaReceivers))
+	if conf.IsSet(metricsDestinationsKey) {
+		exporters := common.NewTranslatorMap[component.Config]()
+		for key := range otherExporters {
+			if _, ok := common.GetString(conf, common.ConfigKey(common.MetricsKey, common.MetricsDestinationsKey, key)); ok {
+				exporters.Set(otherExporters[key])
+			}
+		}
+		if exporters.Len() != 0 {
+			// PRW exporter does not need the delta conversion.
+			receivers := common.NewTranslatorMap[component.Config]()
+			receivers.Merge(hostReceivers)
+			receivers.Merge(deltaReceivers)
+			translators.Set(NewTranslator(fmt.Sprintf("%s_other", common.PipelineNameHost), receivers, exporters))
```

Comment:
What is "%s_other" for?

Reply:
It was meant for all non-cloudwatch destinations, which is an artifact from a previous change. Will make it more specific to AMP.